package/src/specs/RNCandle.nitro.ts

```diff
@@ -126,10 +345,10 @@ export interface RNCandle extends HybridObject<{ ios: "swift" }> {
   initialize(appUser: AppUser): void;
   getLinkedAccounts(): Promise<LinkedAccount[]>;
   unlinkAccount(linkedAccountID: string): Promise<void>;
-  // FIXME: Make this type safe
-  getAssetAccounts(): Promise<string>;
-  getTrades(span: string | undefined): Promise<string>;
-  getTradeQuotes(span: string | undefined): Promise<string>;
+  getAssetAccounts(query: AssetAccountQuery): Promise<AssetAccount[]>;
+  getTrades(query: TradeQuery): Promise<Trade[]>;
+  getTradeQuotes(request: TradeQuoteRequest): Promise<TradeQuote[]>;
+  submitTrade(serviceTradeID: string): Promise<TradeResult>;
```

Comment:
```suggestion
  submitTrade(serviceTradeID: string): Promise<Trade>;
```

#req This will just return a single trade as dicussed

Reply:
I had that before but the codegen failed to compile.